docs/reference/eql/get-async-eql-search-api.asciidoc

```diff
@@ -0,0 +1,83 @@
+[role="xpack"]
+[testenv="basic"]
+
+[[get-async-eql-search-api]]
+=== Get async EQL search API
+++++
+<titleabbrev>Get async EQL search</titleabbrev>
+++++
+
+dev::[]
+
+Returns the current status and available results for an <<eql-search-async,async
+EQL search>> or a <<eql-search-store-sync-eql-search,stored synchronous EQL
+search>>.
+
+[source,console]
+----
+GET /_eql/search/FkpMRkJGS1gzVDRlM3g4ZzMyRGlLbkEaTXlJZHdNT09TU2VTZVBoNDM3cFZMUToxMDM=
+----
+// TEST[skip: no access to search ID]
+
+[[get-async-eql-search-api-request]]
+==== {api-request-title}
+
+`GET /_eql/search/<search_id>`
+
+[[get-async-eql-search-api-prereqs]]
+==== {api-prereq-title}
+
+See <<eql-requirements,EQL requirements>>.
+
+[[get-async-eql-search-api-limitations]]
+===== Limitations
+
+See <<eql-limitations,EQL limitations>>.
+
+[[get-async-eql-search-api-path-params]]
+==== {api-path-parms-title}
+
+`<search_id>`::
+(Required, string)
+Identifier for the search.
++
+A search ID is provided in the <<eql-search-api,EQL search API>>'s response for
+an <<eql-search-async,async search>>. A search ID is also provided if the
+request's <<eql-search-api-keep-on-completion,`keep_on_completion`>> parameter
+is `true`.
+
+[[get-async-eql-search-api-query-params]]
+==== {api-query-parms-title}
+
+`keep_alive`::
+(Optional, <<time-units,time value>>)
+Period for which the search and its results are stored on the cluster. Defaults
+to the `keep_alive` value set by the search's <<eql-search-api,EQL search
+API>> request.
++
+If specified, this parameter sets a new `keep_alive` period for the search,
+starting when the get async EQL search API request executes. This new period
+overwrites the one specified in the EQL search API request.
++
+When this period expires, the search and its results are deleted, even if the
+search is ongoing.
+
+`wait_for_completion_timeout`::
+(Optional, <<time-units,time value>>)
+Timeout duration to block other requests while waiting for the request to
```

Comment:
I am not quite sure what you mean by "block other requests" here. It should only block the current request. Please let me know if you observed other behavior.

Reply:
You're correct. Thanks for catching this. I've updated the definition with 30518af594e34c761a4cb3bdfd0e38c7ced41c48.